nitrocli/src/main.rs

```diff
@@ -49,6 +51,19 @@ const RECV_TRY_COUNT: i8 = 40;
 const SEND_RECV_DELAY_MS: u64 = 200;
 
 
+/// Create an `error::Error` with an error message of the format `msg: err`.
+fn get_error(msg: &str, err: libnitrokey::CommandError) -> Error {
+  let error = format!("{}: {:?}", msg, err);
+  Error::Error(error)
+}
+
+
+/// Connect to a Nitrokey Storage device and return it.
+fn get_storage_device() -> Result<libnitrokey::Storage> {
+  libnitrokey::Storage::connect().or(Err(Error::Error("Nitrokey device not found".to_string())))
```

Comment:
(I understands it's a bit of a pain to identify new problems identified by clippy, but I haven't gotten around cleaning up the existing ones; still, we should avoid introducing new warnings to converge to a maintainable state eventually, even if they are just emitted by clippy and not rustc directly)

```
warning: use of `or` followed by a function call
   --> src/main.rs:105:35
    |
105 |   libnitrokey::Storage::connect().or(Err(Error::Error("Nitrokey device not found".to_string())))
    |                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `or_else(|_| Err(Error::Error("Nitrokey device not found".to_string())))`
```